decidim-core/app/helpers/decidim/scopes_helper.rb

```diff
@@ -14,8 +14,8 @@ module ParticipatoryProcessScopesHelper
     # participatory_process - a Decidim::ParticipatoryProcess.
     #
     # Returns an Array.
-    def participatory_process_subscopes(participatory_process)
-      [Option.new("", I18n.t("decidim.participatory_processes.scopes.global"))] +
+    def subscopes_for(participatory_process)
```

Comment:
I was planning to rename it to `participatory_space` in the other PR (where I introduce the abstraction), but since we've introduced the `Scopable` concern here, what you suggest make sense to me. I'll update it!

Thanks for having a look @leio10! :smiley: